applications/NXmonopd.nxdl.xml

```diff
@@ -27,6 +27,17 @@
     xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
     xsi:schemaLocation="http://definition.nexusformat.org/nxdl/3.1 ../nxdl.xsd"
     >
+    <symbols>
+        <doc>
+            The symbol(s) listed here will be used below to coordinate datasets with the same shape.
+        </doc>
+        <symbol name="i">
+            <doc>Optimum diffracted wavelength</doc>
```

Comment:
I agree the doc string suggests that it shouldn't be a list (rank=1,value=i) but just a single float, it appears that may have been this way in the definition for a while, I will take any advice on what you think it should be as again I am not familiar with this definition

Reply:
I had a look at the base class [NXcrystal](https://manual.nexusformat.org/classes/base_classes/NXcrystal.html) and `i` is the number of wavelengths.

Reply:
@RussBerg any chance of finishing this off soon so I can make a release? Otherwise we can move this to the next milestone.

Reply:
Sorry Peter got bogged down in other work and didn't look to see that you had responded, I will update the doc string to match NXcrsytal so that it says "i is the number of wavelengths"